packages/sources/binance-dex/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,76 @@
+import { AdapterRequest } from '@chainlink/types'
+import request from 'supertest'
+import * as process from 'process'
+import { server as startServer } from '../../src'
+import * as nock from 'nock'
+import * as http from 'http'
+import { mockRateResponseFailure, mockRateResponseSuccess } from './fixtures'
+
+describe('execute', () => {
+  const id = '1'
+  let server: http.Server
+  const req = request('localhost:8080')
+  beforeAll(async () => {
+    process.env.CACHE_ENABLED = 'false'
+    if (process.env.RECORD) {
+      nock.recorder.rec()
+    }
+    server = await startServer()
+  })
+  afterAll((done) => {
+    if (process.env.RECORD) {
+      nock.recorder.play()
+    }
+
+    nock.restore()
+    nock.cleanAll()
+    nock.enableNetConnect()
+    server.close(done)
+  })
+
+  describe('rate api', () => {
+    const data: AdapterRequest = {
+      id,
+      data: {
+        base: 'BUSD-BD1',
+        quote: 'USDT-6D8',
```

Comment:
Ideally this adapter should be able to work with
```
{ base: 'BUSD', quote: 'USDT' }
```

Does that not work?

Reply:
Nope. https://dex-asiapacific.binance.org/api/v1/ticker/24hr?symbol=BUSD_USDT

Reply:
Ok got it, I've added a story to "fix" this adapter. It shouldn't be blocking this PR then